Makefile

```diff
@@ -0,0 +1,61 @@
+DC_RUN = \
```

Comment:
Yep, that's what I was going for -- you save a line for each variable because you can't define Makefile variables on the same line as something else. The downside is that you need to use `$$` rather than `$` due to escaping, but that's not too bad.